autogpt/spinner.py

```diff
@@ -54,12 +54,11 @@ def __exit__(self, exc_type, exc_value, exc_traceback) -> None:
     def update_message(self, new_message, delay=0.1):
         """Update the spinner message
         Args:
-            new_message (str): New message to display
-            delay: Delay in seconds before updating the message
+            new_message (str): New message to display.
+            delay: Delay in seconds before updating the message.
         """
         time.sleep(delay)
-        sys.stdout.write(
-            f"\r{' ' * (len(self.message) + 2)}\r"
-        )  # Clear the current message
+        # Clear the current message
+        sys.stdout.write(f"\r{' ' * (len(self.message) + 2)}\r")
```

Comment:
Revert unless this is required by our linter

Reply:
In the same file, I saw two formats as below, so I adjusted.

        sys.stdout.write(f"\r{' ' * (len(self.message) + 2)}\r")

        sys.stdout.write(
            f"\r{' ' * (len(self.message) + 2)}\r"
        )